lib/polint/parser.rb

```diff
@@ -0,0 +1,52 @@
+require 'parslet'
+
+module Polint
+  class Parser < Parslet::Parser
+
+    rule(:endl) { str("\n").maybe }
+    rule(:sp) { str(' ') }
+    rule(:htab) { str("\t") }
+    rule(:wsp) { sp | htab }
+    rule(:lwsp) { wsp.repeat }
+    rule(:tcase) { match(/[A-Z]/).repeat(1) >> match(/[a-z]/).repeat }
+    rule(:blank_line) { endl }
+
+    rule(:quote) { str('"') }
+    rule(:quoted_char) { match(/[^"]/) }
+    rule(:quoted_pair) { str('\\') >> quote }
```

Comment:
One thing to note here is that as far as I can see the rules for escaping strings in PO files aren't documented anywhere. I've assumed that only `"` needs to be escaped and this works fine on all our files, but I suspect there will be bugs here as we discover more characters that need escaping.